src/test/java/test/integration/ContainerInstallationIntegrationTest.java

```diff
@@ -272,12 +262,11 @@ private void installContainerAssertProfileAndDependencies(final String container
     }
 
     private void executeCmd(String container) throws TimeoutException {
-        shell().execute("arquillian-setup --container-adapter " + container + " --test-framework junit");
+        shell().execute("arquillian-setup --container-adapter " + container + " --test-framework junit", 60);
```

Comment:
Problem is that the runner is `Arquillian.class` so how to add parametrized tests.